src/pages/Wallet/Wallet.jsx

```diff
@@ -15,7 +14,6 @@ const Wallet = ({ amount = "0" }) => {
           <Funds amount={amount} isManage={true} />
         </div>
         <div className="wallet-ctner__contacts">
```

Comment:
container no longer relevant

Reply:
i think this div was suppose to be used when contact content component is done